PP_APP/server/frontend/templates/frontend/upload.html

```diff
@@ -0,0 +1,71 @@
+{% extends "frontend/base.html" %}
+{% load static %}
+{% block content %}
+
+<script>
+    function success() {
+        document.location.href = {% url 'success' %};
+    }
+</script>
+
+<div class="container p-4">
+  <div class="row">
+      <div class="col">
+          <label class="btn">
+              <div class="dropzone">
+                  <h2>TOP VIEW</h2>
+                  <p>(Click OR Drag and Drop)</p>
+              </div>
+              <input type="file" hidden>
+          </label>
+      </div>
+      <div class="col">
+            <label class="btn">
+                <div class="dropzone">
+                    <div class="dztext">
+                      <h2>SIDE VIEW</h2>
+                      <p>(Click OR Drag and Drop)</p>
+                    </div>
+                </div>
+                <input type="file" hidden>
+            </label>
+      </div>
+  </div>
+
+  <div class="row">
+    <div class="col justify-content-center">
+      <button class="btn btn-primary btn-lg" onclick="success()">UPLOAD</button>
+    </div>
+  </div>
+</div>
+
```

Comment:
We'll get rid of the double fields when we start hooking this up to the backend. Not difficult.